app/uk/gov/hmrc/nisp/views/account_mqp.scala.html

```diff
@@ -41,6 +41,8 @@
         'dimension15': '@spSummary.statePensionAge',
         'dimension16': '@spSummary.copeAmount.week',
         'dimension22': '@authenticationProvider',
```

Comment:
I have spoken with Paul Cook about this. At the moment we will keep dimension22 as is, and some point in the future we may remove it and only use dimension38. Don't forget dimention38 is used by PTA and probably by other services that integrate with PTA.